library/rust

```diff
@@ -1,14 +1,14 @@
-# this file is generated via https://github.com/rust-lang-nursery/docker-rust/blob/4bb6465d70c5ff26d58493d4cc575501704dfbb9/generate-stackbrew-library.sh
+# this file is generated via https://github.com/rust-lang-nursery/docker-rust/blob/e840f001dc6f4128b4a3b5add7d03457bab806a9/generate-stackbrew-library.sh
 
 Maintainers: Steven Fackler <sfackler@gmail.com> (@sfackler)
 GitRepo: https://github.com/rust-lang-nursery/docker-rust.git
 
-Tags: 1.23.0-stretch, 1-stretch, 1.23-stretch, stretch, 1.23.0, 1, 1.23, latest
+Tags: 1.24.0-stretch, 1-stretch, 1.24-stretch, stretch, 1.24.0, 1, 1.24, latest
 Architectures: amd64, arm32v7, arm64v8, i386
-GitCommit: e382ab252c238ab77f0b6e45b0705ba94b4e9e1d
-Directory: 1.23.0/stretch
+GitCommit: e840f001dc6f4128b4a3b5add7d03457bab806a9
```

Comment:
These commits should probably be [c481589e552336cffcbe97b6e42a7d1d3e1e9a49](https://github.com/rust-lang-nursery/docker-rust/commit/c481589e552336cffcbe97b6e42a7d1d3e1e9a49). :wink:

Reply:
Oops, not sure what happened there - fixed.